### DISCUSSION TITLE
Arrows to Numeric Keyboard.

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#1638] [Q&A]
I have problems, before I used the "Numeric Keyboard" on tic-80,
but now i am forced to use the arrows, what do i do?

**Comment by bettywolfy** [comment]
hello???

**Comment by bettywolfy** [comment]
hi???

**Comment by RobLoach** [comment]
Hi. It's been two days since you posted this. Have patience.

Could you expand on Numberic keyboard to arrows? How do you expect to use the keyboard, and what is the behavior?

**Reply by bettywolfy** [reply]
Okay, before I used the numeric keypad to play tic80 (you know, the one on the right side of the keyboard), but now I'm forced to use the arrow keys (home, pgup, pgdn and final), but I can get used to the arrows of the keys (it's very difficult), that's it...

(Sorry for my bad English, I'm using google translator.)

**Comment by ChildishGiant** [comment]
Is this related to #1452?
